.travis.yml

```diff
@@ -14,8 +14,11 @@ jobs:
 
     - stage: build
       name: conode
+      language: go
+      go: "1.12.x"
       script:
         - cd conode
+        - go version
```

Comment:
It's already available on the travis build, before running the script, in the `travis_setup_go` call.

Reply:
yes, sorry - debugging stuff leftover...